timers.js

```diff
@@ -1,4 +1,92 @@
+/******************************************
+BONUS VERSION WITH JQUERY
+******************************************/
 
+// how do I change my global variables in init to local variables that everything in the timer object can access?
 
+var timer = {
+  init: function() {
+    stopWatch = $("#timer");
+    reset = $("#reset");
+    start = $("#start");
+    pause = $("#pause");
+    timerStatus = "off";
+    seconds = 0;
+    start.on("click", function () {
+      timer.startTimer();
+    });
+    pause.on("click", function () {
+      timer.pauseTimer();
```

Comment:
Think back to your scope/context/closures class. Is there a way to do this so you don't reference the object variable name?